.gitignore

```diff
@@ -22,3 +22,4 @@ inputs/
 issuesReport.json
 shippable/
 zentaworkaround
+src/main/resources
```

Comment:
This shadows the whole directory out, which is probably not a good idea
I mentioned the resources directory to make it possible to lad the file through classloader, and that should have been test resources anyway.

Reply:
OK.